tests/fb_directaccess/fb_directaccess_test.go

```diff
@@ -0,0 +1,128 @@
+package fbdirectaccess
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/drivers/scheduler/k8s"
+	. "github.com/portworx/torpedo/tests"
+	"os"
+	"testing"
+)
+
+const (
+	// TODO: having the secret key checked in the code probably bad, wonder whats the best way to keep this info?"
+	fbS3AccessKey = "PSFBSAZRDJMJDOMAPHHKBHFOOMIIOCMOEMMGCBPGML"
+	fbS3SecretKey = "85E6126CC4ec2e88c/579e+761CF35E3b8ed4feNKJF"
+	newstackFBUrl = "http://10.16.59.28"
+	establishBackupCredential = "pxctl credentials create --provider s3 --s3-access-key %s --s3-secret-key %s --s3-region us-east-1 --s3-endpoint %s --s3-storage-class STANDARD elitest"
```

Comment:
same here I feel it's so weird to have a const string here for string manipulation, a function would be much better.

Reply:
Yeah, this is a bit of an extreme case where I agree. Also, `elitest` should probably be a constant, and probably changed to something else more explanatory.